plugin/storage/es/options.go

```diff
@@ -206,6 +208,12 @@ func addFlags(flagSet *flag.FlagSet, nsConfig *namespaceConfig) {
 		"(experimental) Use read and write aliases for indices. Use this option with Elasticsearch rollover "+
 			"API. It requires an external component to create aliases before startup and then performing its management. "+
 			"Note that "+nsConfig.namespace+suffixMaxSpanAge+" is not taken into the account and has to be substituted by external component managing read alias.")
+	if nsConfig.namespace == archiveNamespace {
```

Comment:
Just wondering if this needs to be conditional? The primary namespace is enabled by default, so it just means a user could disable the primary namespace, which is unlikely.

Reply:
I don't want to show this flag for the primary NS. We do the same for c*